tests/unit/client_test.py

```diff
@@ -1,15 +1,143 @@
 import os
+
+import sure
```

Comment:
Sure I'll try sure (hah), but I may decide to swap them out later for standard assertions.